diff --git a/pyproject.toml b/pyproject.toml
index b3919a5c..cc2c50bd 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -39,7 +39,7 @@ cryptography = "^43.0.0"
 tzdata = "^2024.1"
 modelscope = "^1.28"
 ray = {version = "2.48.0", extras = ["default"]}
-vllm = {version = "0.10.1.1", optional = true}
+vllm = {version = "0.11.2", optional = true}
 mistral_common = {version = "^1.4.3", optional = true, extras = ["opencv"]}
 uvicorn = "^0.32.0"
 transformers = "^4.51.3"
