diff --git a/gpustack/api/auth.py b/gpustack/api/auth.py
index 9ec7759..a85900c 100644
--- a/gpustack/api/auth.py
+++ b/gpustack/api/auth.py
@@ -152,9 +152,10 @@ async def get_user_from_bearer_token(
 ) -> Optional[User]:
     try:
         parts = bearer_token.credentials.split("_")
-        if len(parts) == 3 and parts[0] == API_KEY_PREFIX:
-            access_key = parts[1]
-            secret_key = parts[2]
+        if len(parts) == 2:
+            # Format: {access_key}_{secret_key}
+            access_key = parts[0]
+            secret_key = parts[1]
             api_key = await APIKeyService(session).get_by_access_key(access_key)
             if (
                 api_key is not None
diff --git a/gpustack/routes/api_keys.py b/gpustack/routes/api_keys.py
index 065855b..6ddd7c0 100644
--- a/gpustack/routes/api_keys.py
+++ b/gpustack/routes/api_keys.py
@@ -8,7 +8,7 @@ from gpustack.api.exceptions import (
     InternalServerErrorException,
     NotFoundException,
 )
-from gpustack.security import API_KEY_PREFIX, get_secret_hash
+from gpustack.security import get_secret_hash
 from gpustack.server.deps import CurrentUserDep, ListParamsDep, SessionDep, EngineDep
 from gpustack.schemas.api_keys import ApiKey, ApiKeyCreate, ApiKeyPublic, ApiKeysPublic
 from gpustack.server.services import APIKeyService
@@ -79,7 +79,7 @@ async def create_api_key(
         name=api_key.name,
         description=api_key.description,
         id=api_key.id,
-        value=f"{API_KEY_PREFIX}_{access_key}_{secret_key}",
+        value=f"{access_key}_{secret_key}",
         created_at=api_key.created_at,
         updated_at=api_key.updated_at,
         expires_at=api_key.expires_at,
